src/components/member-page/donation/table/row-detail.js

```diff
@@ -188,10 +202,14 @@ export const TableRowDetail = memo(({ record, periodicHistory, isLoading }) => {
     is_anonymous: isAnonymous,
     pay_method: payMethod,
     order_number: orderNumber,
+    type,
   } = record
   const receiptAddress = `${record.address_state || ''}${record.address_city ||
     ''}${record.address_detail || ''}`
 
+  const { releaseBranch } = useContext(CoreContext)
+  const editReceiptUrl = `${origins.forClientSideRendering[releaseBranch].support}/contribute/${contributeType[type]}/${orderNumber}/utm_source=supportsuccess&utm_medium=account`
```

Comment:
`utm` should be query string ( `/` -> `?`)

Reply:
fixed